DorisProject/src/app/pages/modal/modal.component.ts

```diff
@@ -0,0 +1,40 @@
+import {Component, Inject, OnInit} from '@angular/core';
+import {MAT_DIALOG_DATA, MatDialogRef} from '@angular/material/dialog';
+import {Router} from '@angular/router';
```

Comment:
Codacy has a fix for the issue: [Replace `Router` with `·Router·`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9269670)
```suggestion
import { Router } from '@angular/router';
```